docs/src/probabilities.md

```diff
@@ -1,10 +1,41 @@
 # [Probabilities](@id probabilities_estimators)
 
+## Probabilities API
+
+```@docs
+ProbabilitiesEstimator
+```
+
+!!! note "A more formal treatment"
+    You may notice that we don't use the word "event" at all in this package. Here's why.
```

Comment:
Honestly, I would vote to remove this section. I have not the slightest clue what a σ-algebra is, and I certainly don't need to understand this package. I think everyone has an intuitive notion of "outcome" as specifying the output of a single sampling from a "random process", or "probability space". So it is immediatelly clear why we should indeed use outcome instead of event. So, to recap, yes for using outcome everywhere, no for including this length mathy section in the docs.

Reply:
Sure, I'm fine with removing this. I think it should be very obvious from the docstring of `ProbabilitiesEstimator` what's going on.